### ISSUE TITLE
[Bug Report] The size of some components is calculated incorrectly in IE11.

### ISSUE THREAD

**Issue by undelalune** [vuetifyjs/vuetify#3031]
### Versions and Environment
**Vuetify:** 1.0.0-beta.4
**Vue:** 2.5.13
**Browsers:** Internet Explorer
**OS:** Windows 10

### Steps to reproduce
IE11: https://next.vuetifyjs.com/components/data-tables#example-editdialog
Click on Editable field Open Edit dialog. 
Dialog is too small. Text is truncated. Compare with Chrome or FF

At the same view select Rows per page: 25
25 is also truncated, but only in IE11

### Expected Behavior
We can see the entire text field

### Actual Behavior
The text is truncated
![chrome](https://user-images.githubusercontent.com/1636423/35008625-22333c30-fb0e-11e7-9904-2625fd55cb11.png)
![ie11](https://user-images.githubusercontent.com/1636423/35008626-224e4f34-fb0e-11e7-83bb-708ecd7ec9f1.png)

**Comment by KaelWD** [comment]
No idea how to fix this tbh, IE just refuses to do what you'd expect. If anyone manages to fix this, please submit a PR.

**Comment by AlexKovynev** [comment]
maybe something from this can help?
https://stackoverflow.com/questions/42510904/inline-flex-item-does-not-grow-with-its-content-in-internet-explorer?rq=1

The default flex behavior of flex items has changed. In Internet   Explorer 10, flex items that didn't fit their containers overflowed   the margins of the container or clipped to the margins of the   container. Starting with IE11, these items now shrink to fit their   containers (up to the min-width value, if specified). Use the   flex-shrink property to change this behavior.   https://msdn.microsoft.com/en-us/library/dn265027(v=vs.85).aspx  

So, the following .cell rules should solve the issue  
.cell {   
flex: 0 0 auto;   
-ms-flex: 0 1 auto; /* overwrites the previous rule only in IE11 */   
border-right: 1px solid black;   
overflow: hidden;   
min-width: 200px; 
}

**Comment by KaelWD** [comment]
```css
.input-group {
  flex: 0 1 auto;
  /* width: 100%; */
}

.input-group__input {
  flex: 0 1 auto;
}
```
Fixes the select in IE, and doesn't seem to break anything in chrome. It does fuck up the edit dialog even worse in IE though.

**Comment by jacekkarczmarczyk** [comment]
Looks fixed in 1.1

**Comment by KaelWD** [comment]
There's a vertical scrollbar now but who cares
![image](https://user-images.githubusercontent.com/16421948/41199028-f0573784-6ccd-11e8-9b63-4769694d3905.png)
